### DISCUSSION TITLE
LoRA generated by sd_dreambooth_extension not working directly in AUTOMATIC1111 webui

### DISCUSSION THREAD

**Discussion by Zarxrax** [d8ahazard/sd_dreambooth_extension#839] [General]
The Automatic1111 webui recently added native support for LoRA files. The .pt files that are generated by the extension show up in the interface, but they have no effect on the generated output image.

I am not sure if this is a limitation of the current support in the webui, or if it is an issue with how the extension is generating the model. Are there any tricks to get it working, or is this just an incompatibility right now?

**Comment by Thomas-MMJ** [comment]
just different file format, there are currently 3 variants of LORAs.  I think you can apply your dreambooth LORA to your model, and then extract it using the compatible alternative format here,

sd-scripts/networks/extract_lora_from_models.py

https://github.com/kohya-ss/sd-scripts
